src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -16,10 +16,6 @@ phasePath: &phasePath ../../phases/query/TimeSeriesLastpoint.yml
 
 # Operations reused in multiple Phases.
 Nop: &Nop {Nop: true}
-Quiesce: &Quiesce
```

Comment:
Shouldn't we also change the Quiesce key and use it (or remove it from the source location).

Reply:
As far as I understand, I've removed the original quiesce definition here:
src/phases/query/TimeSeriesLastpoint.yml

Reply:
Thanks.